flow/aria.js

```diff
@@ -269,6 +269,8 @@ type ARIAPropertyMap = {
 
 type ARIAPropertyDefinition = {
   type: 'string'
+  | 'id',
+  | 'idlist',
```

Comment:
On an interesting note, the [build report](https://travis-ci.org/A11yance/aria-query/jobs/346199294#L848-L851) is passing despite these commas causing parsing errors for the flow check. @jessebeach